source/clientsideapp.md

```diff
@@ -43,26 +43,33 @@ Here we are, doing the traditionnal "Hello World" app that can make you want to
   "displayName": "Your App's Name",
   "version": "1.0.0",
   "description": "Your app's description",
-  "main": "index.html"
+  "icon-path": "main_icon.png",
+  "author": "Your name",
```

Comment:
In my opinion, It is important to have a consistency between documentation examples and github examples.